### ISSUE TITLE
GLArea() error not implemented on OSX

### ISSUE THREAD

**Issue by louisponet** [JuliaGraphics/Gtk.jl#339]
When I try to create a `GLArea()` I get a window that looks like this:
<img width="599" alt="screen shot 2017-12-10 at 12 41 53" src="https://user-images.githubusercontent.com/21249513/33804679-4e910e6e-ddaa-11e7-842f-95d90399f277.png">

This was encountered while trying to incorporate GLVisualize into a Gtk window, see [Integration into Gtk or equivalent](https://github.com/JuliaGL/GLVisualize.jl/issues/195)

**Comment by SimonDanisch** [comment]
@lobingera 
>a libgtk or an Gtk.jl issue?

I have no idea, but the fact that "Not implemented on" can't be found in Gtk.jl at all, indicates that it's a libgtk problem.

**Comment by louisponet** [comment]
Seems like it: 
[gdkglcontext-quartz.c](https://github.com/GNOME/gtk/blob/4724a89022ef1bec93b1a42d4cf2fec7191ed712/gdk/quartz/gdkglcontext-quartz.c)

That's a pity...

**Comment by tknopp** [comment]
Yea its a pity that this does not work. But even if it would: @SimonDanisch Is OpenGL actually useful on OSX these days?

**Comment by lobingera** [comment]
apple thinks so:
https://developer.apple.com/opengl/

**Comment by tknopp** [comment]
Yes but the question is which version is supported (in in particular which shader functionality).

**Comment by louisponet** [comment]
[It seems all relevant are supported?](https://developer.apple.com/opengl/OpenGL-Capabilities-Tables.pdf)

**Comment by SimonDanisch** [comment]
Yes, Apple distances itself from OpenGL more and more in favor of metal, but still supports fairly new versions! So it's definitely useful, while not being very future safe ;)
